packages/sanity/package.json

```diff
@@ -266,8 +266,8 @@
     "yargs": "^17.3.0"
   },
   "devDependencies": {
-    "@playwright/experimental-ct-react": "1.44.1",
-    "@playwright/test": "1.44.1",
+    "@playwright/experimental-ct-react": "1.49.1",
```

Comment:
wonder when it will no longer be experimental 🤔

Reply:
Haha, yes